README.md

```diff
@@ -6,13 +6,13 @@
 ## Table of Contents
 
 1. [Preface](#preface)
-2. [Reference Drivers](#reference-drivers)
+2. [Reference Drivers and Examples](#reference-drivers)
```

Comment:
If you rename it in the TOC wouldn't you also want to rename the title? I used https://github.com/pbzweihander/markdown-toc to generate the TOC originally.

Reply:
@rnestler  Thank you for the input... I've adjusted the titles accordingly.